docker-sync.yml.erb

```diff
@@ -0,0 +1,43 @@
+version: '2'
+options:
+  max_attempt: 20
+  compose-file-path:
+    - 'docker-compose.yml'
+    - 'docker-compose.override.yml'
+  compose-dev-file-path: 'docker-compose-dev-macos.yml'
```

Comment:
Where is the `docker-compose-dev-macos.yml` file? Or am I missing something obvious? :)

Can only find `docker-compose-dev-macos-delegated.yml` and `docker-compose-dev-macos-sync.yml.erb`.

Reply:
Good catch!